qt/aqt/exporting.py

```diff
@@ -38,7 +40,12 @@ def setup(self, did):
         self.frm.format.activated.connect(self.exporterChanged)
         self.exporterChanged(idx)
         # deck list
-        self.decks = [_("All Decks")] + sorted(self.col.decks.allNames())
+        if self.cids:
+            self.decks = [_("Browser's selection")]
```

Comment:
"Browser selection" would be more natural English, but why not just hide the widget completely when called from the Browse screen? Then you wouldn't need to think about the currentIndex() at all in the code below.

Reply:
Pardon my French. 
I'll change it this way. I didn't really give a thought about it and added to the code already existing.